File: src/project/termin.py
#!/usr/bin/env python
# coding: utf-8

import sqlite3

connection = sqlite3.connect("terminKalender.db")
cursor = connection.cursor()

sql_command = """
CREATE TABLE termin(
email VARCHAR(100) PRIMARY KEY,
titel VARCHAR(100),
datum VARCHAR(100),
uhrzeit VARCHAR(100),
zeitspanne VARCHAR(100),
wiederholung VARCHAR(100),
ort VARCHAR(100),
besitzer VARCHAR(100),
beschreibung VARCHAR(1000),
sichtbarkeit VARCHAR(100));"""

cursor.execute(sql_command)
connection.commit()
connection.close()

File: src/project/Termin.java
package project;

import java.sql.Time;
import java.util.Date;



public class Termin{
	private String titel;
	private Date datum;
	private Time uhrzeit;
	private Time zeitspanne;
	private String ort;
	private String besitzer;
	private String wiederholung;
	private String beschreibung;
	private String sichtbarkeit;
	
    public Termin(String titel, Date datum, Time uhrzeit, Time zeitspanne, String ort, String besitzer, String wiederholung,
    String beschreibung, String sichtbarkeit){
        this.titel = titel;
        this.datum = datum;
        this.uhrzeit = uhrzeit;
        this.zeitspanne = zeitspanne;
        this.ort = ort;
        this.besitzer = besitzer;
        this.wiederholung = wiederholung;
        this.beschreibung = beschreibung;
        this.sichtbarkeit = sichtbarkeit;
        
    }

    public void terminLoesches(){

    }

    public void terminAendern(String titel, Date datum, Time uhrzeit, Time zeitspanne, String ort, String besitzer, String wiederholung,
    String beschreibung, String sichtbarkeit){
        this.titel = titel;
        this.datum = datum;
        this.uhrzeit = uhrzeit;
        this.zeitspanne = zeitspanne;
        this.ort = ort;
        this.besitzer = besitzer;
        this.wiederholung = wiederholung;
        this.beschreibung = beschreibung;
        this.sichtbarkeit = sichtbarkeit;
    }

}
